frontend-react/src/components/Admin/EditReceiverSettings.tsx

```diff
@@ -0,0 +1,227 @@
+                />
+                <CheckboxComponent
+                    fieldname={"reverseTheQualityFilter"}
+                    label={"Reverse the Quality Filter"}
+                    defaultvalue={orgReceiverSettings.reverseTheQualityFilter}
+                    savefunc={(v) =>
+                        (orgReceiverSettings.reverseTheQualityFilter = v)
+                    }
+                />
+                <TextAreaComponent
+                    fieldname={"routingFilter"}
+                    label={"Routing Filter"}
+                    defaultvalue={orgReceiverSettings.routingFilter}
+                    savefunc={(v) => (orgReceiverSettings.routingFilter = v)}
+                />
+                <TextAreaComponent
+                    fieldname={"processingModeFilter"}
+                    label={"Processing Mode Filter"}
+                    defaultvalue={orgReceiverSettings.processingModeFilter}
+                    savefunc={(v) =>
+                        (orgReceiverSettings.processingModeFilter = v)
+                    }
+                />
+                <CheckboxComponent
+                    fieldname={"deidentify"}
+                    label={"De-identify"}
+                    defaultvalue={orgReceiverSettings.deidentify}
+                    savefunc={(v) => (orgReceiverSettings.deidentify = v)}
+                />
+                <TextAreaComponent
+                    fieldname={"timing"}
+                    label={"Timing"}
+                    defaultvalue={orgReceiverSettings.timing}
+                    savefunc={(v) => (orgReceiverSettings.timing = v)}
+                />
+                <TextAreaComponent
+                    fieldname={"transport"}
+                    label={"Transport"}
+                    defaultvalue={orgReceiverSettings.transport}
+                    savefunc={(v) => (orgReceiverSettings.transport = v)}
+                />
+                <TextInputComponent
+                    fieldname={"externalName"}
+                    label={"External Name"}
+                    defaultvalue={orgReceiverSettings.externalName}
+                    savefunc={(v) => (orgReceiverSettings.externalName = v)}
+                />
+                <Grid row>
+                    <Button type="button" onClick={() => history.goBack()}>
+                        Cancel
+                    </Button>
+                    <Button
+                        form="edit-setting"
+                        type="submit"
+                        data-testid="submit"
+                        onClick={() => saveRecieverData()}
+                    >
+                        Save
+                    </Button>
+                </Grid>
+            </GridContainer>
+        );
+    };
+
+    return (
+        <NetworkErrorBoundary
+            fallbackComponent={() => <ErrorPage type="page" />}
+        >
+            <section className="grid-container margin-bottom-5">
```

Comment:
I'm a little confused about the `h3` wrapping `FormComponent`? If the goal is for the `span` to be your fallback for the `FormComponent` you should be able to remove `h3`. If you want that `span` to be an `h3`, you can swap the tag and it should still work.

Reply:
I think this is a pattern copied from Daily.tsx. Gitblame says @drew-usds made this change Jan 7th.

Also, the header order in Daily.tsx is <h3><h1> which will trigger a 503 accessibility issue.

I agree we shouldn't propagate it if it's an issue, but we should check with Drew.

Reply:
@kevinhaube the primary effect of removing that outer h3 seems to be that the labels at the top of the page are no longer bold, and further swapping 'span' out for 'h3' has no effect since it doesn't enclose <FormComponent />.

I actually just copied these in yesterday from the AdminMain.tsx page to address a code review suggestion from Sean to make the suspense message look nicer. It was previously much more barebones, perhaps I should just go back to that? Thoughts? @TomNUSDS 

b4
![image](https://user-images.githubusercontent.com/92405130/150593874-3b66ba04-f71f-4597-bb7e-b35a144aa4ae.png)

after
![image](https://user-images.githubusercontent.com/92405130/150594126-8e2f5d43-4427-49b4-89bc-7aadf5922c05.png)

Reply:
@kevinhaube @TomNUSDS never mind all that above. this is resolved now.